pyccl/halos/profiles.py

```diff
@@ -48,6 +48,8 @@ def __init__(self):
                                  'plaw_fourier': -1.5,
                                  'plaw_projected': -1.}
 
+        self.is_number_counts = False
+
```

Comment:
Can we make this a class-level attribute? I.e. the same as `name` in line 38

Reply:
Done